snowpack.config.js

```diff
@@ -1,38 +0,0 @@
-/** @type {import("snowpack").SnowpackUserConfig } */
-module.exports = {
-  env: {
-    RADICLE_ALCHEMY_API_KEY: process.env.RADICLE_ALCHEMY_API_KEY,
-    RADICLE_HTTP_API: process.env.RADICLE_HTTP_API,
-  },
-  mount: {
-    public: '/',
-    src: '/dist',
-  },
-  alias: {
-    '@app': './src',
-  },
-  plugins: [
-    '@snowpack/plugin-svelte',
-    '@snowpack/plugin-typescript',
-  ],
-  routes: [
-    /* Enable an SPA Fallback in development: */
-    {"match": "routes", "src": ".*", "dest": "/index.html"},
-    {"match": "all", "src": "/projects/.*", "dest": "/index.html"},
-    {"match": "all", "src": "/orgs/.*/projects/.*", "dest": "/index.html"},
```

Comment:
This works out of the box? Try a path with a filename in it, eg. `/foobar/file.png` or `/orgs/asd/bar.html` and make sure it routes to the app (should just show a blank screen I think)

Reply:
Yep works out of the box